### ISSUE TITLE
Laravel 6  Call to undefined function Yomo\AddressFactory\kebab_case()

### ISSUE THREAD

**Issue by imjonos** [nonsapiens/addressfactory#5]
Hello
Got this issue 
Call to undefined function Yomo\AddressFactory\kebab_case()

**Comment by casperwilkes** [comment]
I ran into the same issue. I'm using laravel 6.

Did some looking around. It appears that was an old function that changed between 5.6 and 5.7 when they started utilizing facade names.

https://laravel.com/docs/5.6/helpers#method-kebab-case
https://laravel.com/docs/5.7/helpers#method-kebab-case

@yomo needs to update the code to use the `Str::kebab` method and the `Arr::random` method.

Line: 8
```php
use Illuminate\Support\{Arr, Str};
```

Line: 139
```php
if ( $cnfCountry = Config::get( 'realaddress.' . Str::kebab( $country ), false ) ) {
```

Line: 146
```php
$query = implode( ', ', [ Arr::random( $locations ), $country ] );
```

Edit: Removed incorrect information. I had an API key, but did not have google account setup correctly for billing, so my key was being denied.
